pytorch_toolkit/object_detection/requirements.txt

```diff
@@ -3,3 +3,4 @@
 networkx==2.3
 opencv-python==4.1.0.25
 tensorboardX==1.8
+Polygon3==3.0.8
```

Comment:
this is already in mmdetection/requirenments/runtime.txt that is installed before see 2nd line of this file